### ISSUE TITLE
Serializer returns OrderedDict() instead of primitive one on djongo DictField/ListField

### ISSUE THREAD

**Issue by Skyine918** [SomeoneInParticular/rest_meets_djongo#1]
_version: rest-meets-djongo==0.0.10_
When model field is of type DictField (and Serializer with **many=True**) or ListField with many MongoDB Objects (dictionaries), serializers return OrderedDict instead of normal json 'dict'.
Code to reproduce:
**Model:**
```
from djongo import models
class TestModel(models.Model):
    field1 = models.CharField(max_length=16, blank=False)
    field2 = models.CharField(max_length=16, blank=False)
    field3 = models.CharField(max_length=16, blank=True, default='')

    problem = models.DictField()
```
**Serializer:**
```class TestModelSerializer(serializers.DjongoModelSerializer):
    class Meta:
        model = TestModel
        fields = ["id", "field1", "field2", "field3", "problem"]
```
**View:**
```
@api_view(['GET'])
def get_all_test_models(request):
    tests = TestModel.objects.all()
    tests_serializer = TestModelSerializer(tests, many=True)
    from rest_framework.response import Response
    from rest_framework import status
    return Response(data=tests_serializer.data, status=status.HTTP_200_OK)
```
**Response**
```{
  "id": 1,
  "field1": "QWE",
  "field2": "1231231",
  "field3": "123123123123",
  "problem": "OrderedDict([('field', 'aaa')])"
}
```

**Comment by SomeoneInParticular** [comment]
Hello; I just pushed an update which should hopefully fix that flub (v0.0.11). It should also help alleviate some other oddities with Dictionary and List Fields; let me know if something isn't working as intended regarding those.

One quick note regarding ListFields; currently they rely on DRF's ListField, which does not support empty lists very well. I'm working on bypassing this now.

Cheers!

**Comment by Skyine918** [comment]
I have tested DictField in Serializer with many=True, and List field with many MongoObjects inside, and everything works as intended. Thank you for quick fix and for the library =)
